scripts/install_debs.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+parent_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+
+cd "$parent_path"
+
+bash ./build_debs.sh
+
+dpkg -i ../dist/*.deb
```

Comment:
`dpkg` on its own doesn't install dependencies (it will fail if they're not already installed), so it would be convenient to use `apt` which does. This works on both Ubuntu and Debian.
```suggestion
sudo apt install ../dist/*.deb
```

Reply:
fixed